app/assets/javascripts/ui_components.js.erb

```diff
@@ -2,17 +2,17 @@
 //= require jquery.turbolinks
 //= require jquery_ujs
 //= require underscore
-//= require moment
+//= require moment/moment
 //= require turbolinks
 
-//= require jquery.floatThead/jquery.floatThead
+//= require jquery.floatThead/dist/jquery.floatThead
 
-//= require chosen
-//= require ajax-chosen
+//= require chosen/chosen.jquery
+//= require ajax-chosen/lib/ajax-chosen.min
```

Comment:
I would prefer the non-minified versions if they are available. It leads to easier debugging and better compression under some circumstances.